README.rst

```diff
@@ -262,14 +264,14 @@ Login with:
 
     password: ``test``
 
-The icons are picked from `Feather`_. Many thanks to them for the good work.
+The icons are picked from Feather_. Many thanks to them for the good work.
 
-.. _`Feather`: https://feathericons.com
+.. _Feather: https://feathericons.com
 
 
 Contributing
 ============
 
-For contributing, please see the instructions at `Contributing_`
+For contributing, please see the instructions at Contributing_
 
-.. _`Contributing`: './CONTRIBUTING.rst`
+.. _Contributing: https://github.com/Radi85/Comment/blob/master/CONTRIBUTING.rst
```

Comment:
The relative link would work for both read the doc and github. this will always take them to github.

Reply:
you may be right. But it doesn't currently in the master

Reply:
it was because of syntax errors i made when referencing the link. it should work now if you do so.

Reply:
The full link would be useful when using readme in the docs.

Reply:
> The full link would be useful when using readme in the docs.

I'm not sure i completely understand this logic.

I just though it would be a bit weird to redirect people to github from read the doc when the file is obviously present there. but this shouldn't be too much of an issue. feel free to ignore and resolve this if you want.